src/test/java/com/egov/business-services/tests/collection-services.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Business Services - Collection service tests
+
+ Background:
+     * def jsUtils = read('classpath:jsUtils.js')
+     * configure headers = read('classpath:websCommonHeaders.js')
```

Comment:
as this is already present in pretest. just remove and check it should work fine.

Reply:
@pundaleek-egov Comment have been addressed